experimental/PropertyDDS/packages/property-changeset/src/test/validator/templateValidator.spec.ts

```diff
@@ -14,6 +14,7 @@
  * @fileoverview In this file, we will test template validation.
  */
 
+import { assert } from "@fluidframework/core-utils";
```

Comment:
Since this is test code, you can use node's assert. The one in core-utils is only needed for shipping code. The reason we use our own asserts (Well there are probably multiple) is that we "tag" them, where we remove their message string and replace it with a numeric value. We don't want or need that in test code, so you can avoid the new dependency.

```suggestion
import assert from "node:assert/strict";
```